src/vorta/scheduler.py

```diff
@@ -333,24 +333,29 @@ def reload_all_timers(self):
             self.set_timer_for_profile(profile.id)
 
     def next_job(self):
-        next_job = now = dt.now()
-        next_profile = None
-        for profile_id, timer in self.timers.items():
-            if timer['type'] != ScheduleStatusType.SCHEDULED:
-                continue
-
-            if next_job == now and timer['dt'] > next_job and timer['qtt'].isActive():
-                next_job = timer['dt']
-                next_profile = profile_id
-            elif next_job > now and timer['dt'] < next_job and timer['qtt'].isActive():
-                next_job = timer['dt']
-                next_profile = profile_id
-
-        if next_profile is not None:
-            profile = BackupProfileModel.get_or_none(id=next_profile)
-            return f"{next_job.strftime('%H:%M')} ({profile.name})"
-        else:
-            return self.tr('None scheduled')
+        now = dt.now()
+
+        def is_scheduled(timer):
+            return (
+                timer["type"] == ScheduleStatusType.SCHEDULED
+                and timer["qtt"].isActive()
+                and timer["dt"] >= now
+            )
+
+        scheduled = {profile_id: timer for profile_id, timer in self.timers.items() if is_scheduled(timer)}
+        if len(scheduled) == 0:
+            return self.tr("None scheduled")
+
+        closest_job = min(scheduled.items(), key=lambda item: item[1]["dt"])
+        profile_id, timer = closest_job
+        next_job_dt = timer["dt"]
```

Comment:
> time a better name than next_job_dt 

i didn't like how the variable was with the same name as the enclosing function, so just added a suffix.

> This can be reduced

thought it was clearer than putting everything in the same line. you decide

Reply:
> thought it was clearer than putting everything in the same line. you decide

I am one of those programmers who like one-liners.

Reply:
> i didn't like how the variable was with the same name as the enclosing function, so just added a suffix.

This is Python. Class methods are accessed through `self`. So there is no problem in using the same name as a local variable.

Reply:
i know it's not an issue here, i just don't like using the same name for something else